x/delayedack/types/key_rollapp_packet.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"encoding/binary"
+	fmt "fmt"
+
+	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
+)
+
+var _ binary.ByteOrder
+
+const (
+	// RollappPacketKeyPrefix is the prefix to retrieve all RollappPackets
+	RollappPacketKeyPrefix = "RollappPacket/value/"
+)
+
+// GetRollappPacketKey constructs a key for a specific RollappPacket
+func GetRollappPacketKey(
```

Comment:
why u use height in the key path? it's not unique
why not using just the packet's `Sequence`?

Reply:
I need to iterate and get the relevant packets by heights in an efficient way once it's related state is finalized (look at `ListRollappPackets`). 
the overall rollapp packet key is unique as it also combines sequence and channel.